.travis.yml

```diff
@@ -21,12 +21,13 @@ script:
   - cibuildwheel --output-dir wheelhouse
   - $PIP install -r requirements.txt
   - $PIP install -r dev-requirements.txt
-  - tox -e codecov
+  - wheel=`ls ${wheelhouse}/psautohint*.whl | head -n 1`
+  - tox --installpkg $wheel -e codecov --workdir {project} -c {project}
```

Comment:
we don't want to run tox ourselves again, after cibuildwheel. We want the `CIBW_TEST_COMMAND` to call tox with the --installpkg option...
cibuildwheel builds the wheel, then installs it and runs the given test command. We want tox to take care of configuring the test environment, but we want to make it use a pre-build package (the wheel that we just built with cibuildwheel), instead of rebuilding the module from a sdist (which is its default behavior).
I need to go now, I'll have a look again tomorrow.